dist/main.bundle.js

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
You followed the proper structure for a webpack project with ease 😉 Great job!

Reply:
Webpack is hard to understand but once you do, it is like a walk in the park.